test/664-raceway.py

```diff
@@ -0,0 +1,10 @@
+# https://www.openstreetmap.org/way/28825404
+assert_has_feature(
+    16, 10476, 25242, 'roads',
+    { 'id': 28825404, 'kind': 'minor_road', 'highway': 'raceway' })
```

Comment:
I'm fine with brittle for now. File a new issue for the generative tests for the future?